db/migrate/20230405140343_add_buyer_livin_value_check.rb

```diff
@@ -0,0 +1,5 @@
+class AddBuyerLivinValueCheck < ActiveRecord::Migration[7.0]
+  def change
+    add_column :sales_logs, :buyer_livein_value_check, :integer
```

Comment:
similar to other comments I'm a little confused as to how this copes with buyer 1 and buyer 2, I would expect them to need two independent fields in case both are triggered and only one has been dealt with by the user.

Reply:
I think we do the same thing for all the validations for different people, it would work as expected, but if you've answered this check before as yes, it would have it preselected, so you'd have to either change or confirm again. It's maybe not the most expected behaviour but the check would still show up for both.
It avoids adding a column for each check, plus with soft validations redesign coming up, we will change the flow anyway

Reply:
ok cool. I think I can imagine a case where it wouldn't make sense in the current setup but it's edge enough to probably not worry about.
1. trigger soft validation on buyer 1 with invalid answer
2. don't answer soft validation, just go back
3. go to question related to buyer 2 and input valid answer
4. see soft validation despite valid answer?

Reply:
Hm, I think it wouldn't route to that if the second buyer answer is valid. Cause it would be a different soft validation